src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseRepository.cs

```diff
@@ -49,12 +49,24 @@ public class ReleaseRepository : IReleaseRepository
                 .Where(r => r.UserId == userId && r.Role != ReleaseRole.PrereleaseViewer)
                 .Select(r => r.ReleaseId)
                 .ToListAsync();
-            
+
+            var userPublicationIds = await _contentDbContext
+                .UserPublicationRoles
+                .Where(r => r.UserId == userId && r.Role == PublicationRole.Owner)
+                .Select(r => r.PublicationId)
+                .ToListAsync();
+            var userPublicationRoleReleaseIds = await _contentDbContext
+                .Releases
+                .Where(r => userPublicationIds.Contains(r.PublicationId))
+                .Select(r => r.Id)
+                .ToListAsync();
+            userReleaseIds.AddRange(userPublicationRoleReleaseIds);
```

Comment:
Is there any chance that a release could appear twice in this list? E.g included because they have had contributor role on a release which hasn't been removed before receiving publication owner role on the publication?

Reply:
I think there is, but I assumed it wasn't an issue - I'll investigate.

Reply:
I've made the elements of userReleaseIds distinct. I don't think it affects the code as it currently works, but I can imagine future bugs where this causes and issue so this is probably for the best.